configure.ac

```diff
@@ -72,8 +72,9 @@ EMER_REQUIRED_MODULES="$EOS_METRICS_REQUIREMENT \
                        $SOUP_REQUIREMENT \
                        $UUID_REQUIREMENT"
 AC_SUBST(EMER_REQUIRED_MODULES)
+
 # This indicates a required dbus mocking interface for unit testing.
-AX_PYTHON_MODULE_VERSION([dbusmock], 0.10)
+AX_PYTHON_MODULE_VERSION([dbusmock], 0.10, python3)
```

Comment:
This needs to move down in order to pick up the results of `AC_PATH_PROG` for Python. It should probably go in the "Required libraries" section, so it was in the wrong place anyway.